### ISSUE TITLE
Allow numeric-only article slugs again?

### ISSUE THREAD

**Issue by citxx** [django-wiki/django-wiki#878]
I see, that the were explicitly forbidden in #558, because they were causing some errors, but is it possible to go around these errors without forbidding numeric-only slugs?

There are some quite legit use cases for this, like using a year number as a slug

**Comment by benjaoming** [comment]
It's perhaps possible that only the root slug isn't allowed to be a number.

**Comment by citxx** [comment]
Yes, looks like there should be no url conflicts for non-root article. But currently all the slugs are not allowed to be numbers.

**Comment by teury** [comment]
Now I have noticed it.

It can be corrected when editing these lines.
https://github.com/django-wiki/django-wiki/blob/dd09cabc9419c86aa2d661ef04ac25b1b1dabea7/src/wiki/views/article.py#L72
